examples/resources/stackit_kms_key/resource.tf

```
@@ -0,0 +1,10 @@
+resource "stackit_kms_key" "name" {
+  algorithm    = "example algorithm"
+  backend      = "software"
+  description  = "new descr"
+  display_name = "example name"
+  import_only  = false
+  key_ring_id  = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+  project_id   = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+  purpose      = "example purpose"
```

We usually have the `project_id` and depending ids, like in this case `key_ring_id` at the top of the example. And for the basic example we only set the required fields.
```suggestion
  project_id   = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
  key_ring_id  = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
  algorithm    = "example algorithm"
  backend      = "software"
  description  = "example description"
  display_name = "example name"
  purpose      = "example purpose"
```

If you want you can also add an additional example, where the `stackit_kms_key` resource is used together with `stackit_kms_key_ring`, but this is not necessary. If you add it, please add a comment to indicate, that it's a different example